pifx/util.py

```diff
@@ -52,3 +53,7 @@ def handle_error(response):
         raise Exception(raise_error)
     else:
         return True
+def encode_url_path(url):
+    """Encodes the path url string replacing special characters with properly escaped sequences.  
+    Not intended for use with query string parameters. """
+    return urllib.quote(url)
```

Comment:
You should use `six.moves.urllib.parse.quote()`. Perhaps we do need `six` after all?